wdl-format/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.1.0 - 07-17-2024
```

Comment:
Should this be `# Unreleased` until we actually cut a release?

Reply:
I think so. I'll change it to unreleased.